src/js/components/AdminPanels.js

```diff
@@ -4,28 +4,16 @@ import { useModal, ConfirmationModal, ModalProvider } from "./Modal"
 import { simplePost, simpleGet } from "../modules/utils"
 import PropTypes from "prop-types"
 
-// Utility function to fetch and update menus
-const fetchAndUpdateMenus = (redisEndpointUrl, menus, setMenus) => {
-  simpleGet(redisEndpointUrl, { keys: menus.map((menu) => menu.key) }).then(
-    (dataStr) => {
-      const data = JSON.parse(dataStr)
-      const updatedMenus = menus.map((menu) => {
-        const value = data[menu.key]
-        const selectedItem = menu.items.find((item) => item.value === value)
-        return { ...menu, selectedItem }
-      })
-      setMenus(updatedMenus)
-    }
-  )
-}
-
 // Custom hook to handle Redis post success/failure
 function useRedisStatus() {
   const [redisChanged, setRedisChanged] = useState(null)
 
   const updateRedisStatus = (status) => {
     setRedisChanged(status)
-    setTimeout(() => setRedisChanged(null), 2000)
+    // Only auto-clear if status is "true" or "false"
+    if (status === "true" || status === "false") {
+      setTimeout(() => setRedisChanged(null), 2000)
+    }
```

Comment:
Is this timeout necessary?

Reply:
Yes, that's the slow fadeout of the status indicator. It's best if it clears after an interval so that it's back to the default before another value is inputted.